odm2api/ODM2/models.py

```diff
@@ -1019,7 +734,10 @@ class Specimens(Base):
     SpecimenMediumCV = Column('specimenmediumcv', ForeignKey(CVMediumType.Name), nullable=False, index=True)
     IsFieldSpecimen = Column('isfieldspecimen', Boolean, nullable=False)
 
-    SamplingFeatureObj = relationship(SamplingFeatures)
+    # SamplingFeatureObj = relationship(SamplingFeatures)
+    __mapper_args__ = {
+        'polymorphic_identity':'Specimen',
```

Comment:
Should it be `'polymorphic_identity':'specimens',` (lowercase and plural)? I don't know myself, but that seems more consistent with what you're doing in `Results`